src/YieldDisburser.sol

```diff
@@ -106,32 +109,30 @@ contract YieldDisburser is OwnableUpgradeable {
         uint256 _blockTime,
         uint256 _minVotingAmount,
         uint256 _minHoldingDuration,
-        uint256 _maxVotes,
         uint256 _maxPoints,
-        uint48 _minTimeBetweenClaims,
+        uint256 _cycleLength,
         uint48 _lastClaimedTimestamp,
         uint256 _lastClaimedBlockNumber,
         uint256 _precision
     ) public initializer {
-        __Ownable_init(msg.sender);
-
         BREAD = Bread(breadAddress);
```

Comment:
not sure why this whole section was refactored. it was specifically formatted to be:

initializers
"constant" initializers
public initializers, with projects broken into it's own section because there was a necessary logic block

Reply:
fixed with b367c79 and others